addons/web_editor/static/lib/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -183,7 +184,15 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false)
     if (cursorNode.nodeType !== Node.TEXT_NODE) {
         const { cType } = getState(cursorNode, cursorOffset, DIRECTIONS.LEFT);
         if (cType & CTGROUPS.BLOCK && (!alreadyMoved || cType === CTYPES.BLOCK_OUTSIDE)) {
-            cursorNode.oDeleteBackward(cursorOffset, alreadyMoved);
+            cursorNode.oDeleteBackward(cursorOffset, alreadyMoved, firstBlockIndex - offset);
+        } else if (!alreadyMoved) {
+            const cursorNodeNode = cursorNode.childNodes[cursorOffset];
+            const cursorNodeRightNode = cursorNodeNode ? cursorNodeNode.nextSibling : undefined;
+            if (cursorNodeRightNode &&
+                cursorNodeRightNode.nodeType === Node.TEXT_NODE &&
+                nextSibblings === cursorNodeRightNode) {
+                moveDest[0].insertBefore(document.createElement('br'), cursorNodeRightNode);
```

Comment:
Comment please.
So if we're in the middle of text we add a br instead of deleting? I'm confused :-P